docs/getting_started/overview.rst

```diff
@@ -0,0 +1,53 @@
+************
+Overview
+************
+
+The ``trifinger_simulation`` package provides an interface to the TriFinger robots
+in simulation using the `pybullet physics engine <https://pypi.org/project/pybullet/>`_.
+
+.. note::
+    The TriFinger robots are a little family of kinematically similar robots, including the
+    TriFingerOne, the TriFingerEdu, and the TriFingerPro. These robots signify the different iterations
+    towards making a low-cost, robust, reproducible, and open-source robotic platform, which is both durable
+    against the heavy wear-and-tear of exploration-heavy reinforcement learning algorithms, and also allows
+    for precise control. Among these three, the TriFingerEdu is completely open-source and can be built from
+    scratch. For details, please refer `here <https://github.com/open-dynamic-robot-initiative/open_robot_actuator_hardware/blob/master/mechanics/tri_finger_edu_v1/README.md>`_.
+    You can also find other details on this platform in our `preprint <https://arxiv.org/abs/2008.03596>`_.
+
+    .. list-table:: 
```

Comment:
There seems to be something wrong with this table.  In the rendered documentation it shows up as a bullet-point list with two bullet points on each entry.
Actually, since there is only one column, wouldn't it work to just put the figures one after another without any table or list?